test/integration/targets/win_scheduled_task/tasks/tests.yml

```diff
@@ -211,3 +211,124 @@
     that:
     - remove_scheduled_task_new_path_1.msg == 'Task does not exist'
   when: in_check_mode
+
+
+# Test scheduled task RunAs and RunLevel options
+
+- name: Remove potentially leftover run options task 1
+  win_scheduled_task: &wstr1_absent
```

Comment:
It's a quick reference for storing the values needed for the removal of the scheduled task, i.e. without the unneeded options for when it's present, and/or having to specify it twice (at the beginning, and end).